src/cli.nim

```diff
@@ -58,6 +58,11 @@ proc prefix(kind: CmdLineKind): string =
   of cmdLongOption: "--"
   of cmdArgument, cmdEnd: ""
 
+proc showErrorForMissingVal(kind: CmdLineKind, key: string, val: string) =
```

Comment:
Maybe update the name a bit to indicate that the error is shown conditionally?

```suggestion
proc showErrorIfMissingVal(kind: CmdLineKind, key: string, val: string) =
```

Reply:
(I've now removed this proc).